imperative/python/megengine/functional/tensor.py

```diff
@@ -148,33 +148,30 @@ def full(
     return broadcast_to(x, shape)
 
 
-def ones(shape, dtype="float32", device=None) -> Tensor:
-    r"""Returns a ones tensor with given shape.
+def ones(
+    shape: Union[int, Tuple[int, ...]], 
+    dtype="float32", 
+    device: Optional[CompNode] = None
+) -> Tensor:
+    r"""Returns a tensor with given shape filled with value one.
 
     Args:
-        shape: a list, tuple or integer defining the shape of the output tensor.
-        dtype: the desired data type of the output tensor. Default: ``float32``.
-        device: the desired device of the output tensor. Default: if ``None``,
+        shape (list, tuple, integer): the shape of the output tensor.
```

Comment:
我建议 `shape` 参数的 type hint 用 numpy 的表述进行覆盖，

```suggestion
        shape (int or sequence of ints): the shape of the output tensor.
```

不同类型的 `shape` 输入可以在 Example 中体现。